screenpipe-vision/src/core.rs

```diff
@@ -334,3 +416,37 @@ pub fn get_apple_languages(languages: Vec<screenpipe_core::Language>) -> Vec<Str
         .filter_map(|lang| map.get(lang).map(|&s| s.to_string()))
         .collect()
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum RealtimeVisionEvent {
+    WindowOcr(WindowOcr),
+    UIFrame(UIFrame),
```

Comment:
so confused by why we have such different naming / variables for OCR vs UI

so inconsistent 

also why UI does not have image and less fields stuff